english/light-fire/eating/stream/scream/scream.md

```diff
@@ -0,0 +1,5 @@
+The tower cannot hear you scream, for it does not have any ears. Or maybe you're just too far away.
+
+[Go fish](go-fish/fish.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.